src/codec.rs

```diff
@@ -162,7 +162,7 @@ mod test {
     use ssh_key::{PrivateKey, PublicKey};
     use std::io::Cursor;
 
-    pub fn reader(data: &'static [u8]) -> Cursor<&[u8]> {
+    pub fn reader(data: &'static [u8]) -> Cursor<&'static [u8]> {
```

Comment:
modified because of compiler warning

Reply:
This has been resolved in a separate diff